modal/README.md

```diff
@@ -37,7 +37,7 @@ Modal does not include any events.
 
 ### Metrics
 
-There are no metrics collected for this integration.
+See [metadata.csv][5] for a list of metrics included with this integration.
```

Comment:
Not nitpicking your writing here - I think we need to use exact wording to ensure the "magic phrase" works properly:
```suggestion
See [metadata.csv][5] for a list of metrics provided by this integration.
```